Misc/NEWS.d/next/Tests/2023-05-15-02-22-44.gh-issue-104494.Bkrbfn.rst

```diff
@@ -0,0 +1,2 @@
+Update test_pack_configure_in and test_place_configure_in
```

Comment:
```suggestion
Update Tkinter tests
```

...or, just adjust the formatting:

```suggestion
Update ``test_pack_configure_in`` and ``test_place_configure_in``
```

Reply:
Actually, I prefer my first suggestion. Please apply that.

Reply:
I committed 2nd option even if it looks otherwise.